pkg/workflows/wasm/host/standard_tests/internal/rawsdk/helpers_wasip1.go

```diff
@@ -0,0 +1,215 @@
+package rawsdk
```

Comment:
@nolag I'm not super keen on this -- I think this will just create a situation where the reference sdk will diverge from the ones we actually use

What are you trying to achieve by doing this?